tests/rolling-upgrade-gce.yaml

```diff
@@ -10,6 +10,7 @@ prepare_write_stress: cassandra-stress write no-warmup cl=QUORUM n=10000000 -sch
 stress_cmd_read_10m: cassandra-stress read no-warmup cl=QUORUM duration=10m -schema 'replication(factor=3) compression=SnappyCompressor' -port jmx=6868 -mode cql3 native compression=snappy -rate threads=1000 -pop seq=1..10000000 -log interval=5
 stress_cmd_read_clall: cassandra-stress read no-warmup cl=ALL n=10000000 -schema 'replication(factor=3) compression=DeflateCompressor' -port jmx=6868 -mode cql3 native compression=none -rate threads=1000 -pop seq=1..10000000 -log interval=5
 stress_cmd_read_20m: cassandra-stress read no-warmup cl=QUORUM duration=20m -schema 'replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..10000000 -log interval=5
+stress_cmd_sst3: cassandra-stress user profile=data_dir/sst3_schema.yaml  ops'(insert=3,read1=1,read2=1,update_static=1,update_ttl=1,update_diff1_ts=1,update_diff2_ts=1,update_same1_ts=1,update_same2_ts=1,alter_table=1,delete_row=0.1)' cl=QUORUM duration=20m
```

Comment:
We should have prepare that writes n= rows.
Then, we should have read that verify them.

Reply:
fixed.